go.mod

```diff
@@ -8,7 +8,9 @@ require (
 	github.com/rs/zerolog v1.18.0
 	github.com/stretchr/testify v1.5.1
 	go.dedis.ch/kyber/v3 v3.0.12
+	golang.org/x/lint v0.0.0-20190313153728-d0100b6bd8b3 // indirect
```

Comment:
Do those lines disappear with a `go mod tidy` ? My Go tools remove them all the time but I'm using 1.14 so they might have change something from 1.13

Reply:
Updating from 1.12 to 1.14 solved the problem 👍